src/App.scss

```diff
@@ -1,3 +1,18 @@
+// Colors
+$info: #5bc0de;
+$group-1: #E23C3C;
```

Comment:
Please change the names to include the color and how it's being used. It can be `$purple` but it could also be `$purple-group-color`. You don't want to make it too specific unless it's only used for one purpose, but I think the later is semantic.